backend/api/Startup.cs

```diff
@@ -93,17 +92,32 @@ public void ConfigureServices(IServiceCollection services)
                     member.Type.IsGenericType &&
                     member.Type.GetGenericTypeDefinition() == typeof(ICollection<>));
             });
+            services.Configure<JsonSerializerOptions>(this.Configuration.GetSection("Serialization:Json"));
```

Comment:
Adding more configuration options to support serialization for making HTTP requests.